lib/setup/SetTime.dart

```diff
@@ -56,16 +56,11 @@ class _SetTimeState extends State<SetTime> {
                     onPressed: () => showPresetOptions())),
             // the accept and cancel buttons
             ButtonBar(
-              alignment: MainAxisAlignment.center,
+              alignment: MainAxisAlignment.end,
               mainAxisSize: MainAxisSize.max,
               buttonPadding: EdgeInsets.fromLTRB(30, 10, 30, 10),
               children: <Widget>[
                 FlatButton(
-                    child: Text("Cancel", style: TextStyle(fontSize: 20)),
```

Comment:
Mixed feelings on taking out cancel and exit but it doesn't change the functionality of the app, just makes the flow more specific. If this is the way you think we should go, let's do it